oap-server/server-starter/src/main/resources/ui-initialized-templates/postgresql/postgresql-service.json

```diff
@@ -0,0 +1,700 @@
+               "metrics":[
+                  "meter_pg_fetched_rows_rate"
+               ],
+               "graph":{
+                  "type":"Area",
+                  "opacity":0.4,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Fetched Rows Trend"
+               },
+               "value":"9",
+               "label":"9"
+            },
+            {
+               "x":6,
+               "y":12,
+               "w":6,
+               "h":10,
+               "i":"10",
+               "type":"Widget",
+               "metricTypes":[
+                  "readLabeledMetricsValues"
+               ],
+               "metrics":[
+                  "meter_pg_inserted_rows_rate"
+               ],
+               "graph":{
+                  "type":"Area",
+                  "opacity":0.4,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Inserted Rows Trend"
+               },
+               "value":"10",
+               "label":"10"
+            },
+            {
+               "x":6,
+               "y":32,
+               "w":6,
+               "h":10,
+               "i":"11",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValues",
+                  "readMetricsValues",
+                  "readMetricsValues",
+                  "readMetricsValues",
+                  "readMetricsValues"
+               ],
+               "metrics":[
+                  "meter_pg_buffers_clean",
+                  "meter_pg_buffers_backend",
+                  "meter_pg_buffers_checkpoint",
+                  "meter_pg_buffers_backend_fsync",
+                  "meter_pg_buffers_alloc"
+               ],
+               "graph":{
+                  "type":"Line",
+                  "step":false,
+                  "smooth":false,
+                  "showSymbol":false,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Buffers Trend"
+               },
+               "value":"11",
+               "label":"11"
+            },
+            {
+               "x":12,
+               "y":12,
+               "w":6,
+               "h":10,
+               "i":"12",
+               "type":"Widget",
+               "metricTypes":[
+                  "readLabeledMetricsValues"
+               ],
+               "metrics":[
+                  "meter_pg_updated_rows_rate"
+               ],
+               "graph":{
+                  "type":"Line",
+                  "step":false,
+                  "smooth":false,
+                  "showSymbol":false,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Updated Rows Trend"
```

Comment:
Suggest adding a unit or specific explanation for the `Trend`, same as the other place.

Reply:
I get it.